src/convertToObject.js

```diff
@@ -6,7 +6,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesArray = sourceString.trim().split(';');
+
+  const stylesObject = {};
```

Comment:
The variable name 'stylesObject' is too generic. A descriptive name like 'styleRules' or 'cssProperties' would enhance the readability of the code.

Reply:
> The variable name 'stylesObject' is too generic. A descriptive name like 'styleRules' or 'cssProperties' would enhance the readability of the code.

Good point